core/block_svg.js

```diff
@@ -1035,13 +1038,12 @@ Blockly.BlockSvg.prototype.disposeUiEffect = function() {
   clone.bBox_ = clone.getBBox();
   // Start the animation.
   clone.startDate_ = new Date();
-  Blockly.BlockSvg.disposeUiStep_(clone, this.RTL);
+  Blockly.BlockSvg.disposeUiStep_.bind(this)(clone, this.RTL);
```

Comment:
Indeed, I'd just say:
Blockly.BlockSvg.disposeUiStep_.call(this, clone, this.RTL);
No need to create a new function object with each call.